proposals/solid-oidc/index.bs

```diff
@@ -261,7 +261,7 @@ claim redundant, so in Solid-OIDC the `aud` claim MUST be a string with the valu
 `cnf` — For all flows that require DPoP, the confirmation claim is REQUIRED, as per
 [DPoP Internet-Draft](https://tools.ietf.org/html/draft-fett-oauth-dpop-04#section-7) specification.
 
-`client_id` — REQUIRED in all flows except OIDC registration.
+`client_id` — REQUIRED in all flows except OIDC Dynamic Client registration. See [[!OpenIDConnectDynamicClientRegistration]].
```

Comment:
You are correct that we should clarify the language here. According to section 5, there are three categories of client identifiers:
 1. Remote Client WebID URLs
 2. Public Identifiers (`solid:PublicOidcClient`)
 3. Static and dynamic OIDC/Auth2 client registration

And according to that section, they all need to present a `client_id` value. Admittedly, the third category will likely be less useful, though it may still be important to distinguish between those clients and clients in the second category.

Rather than excluding the third category from this requirement, there is a simplicity to just requiring this claim in all access tokens for Solid. Plus, it will also be a critical part of the authZ component for any Solid server.

Reply:
I'll ammend my pull request to reflect the fact that the `client_id` claim is REQUIRED and conforms to section 5.